platforms-wg/glossary/_index.md

```diff
@@ -1,43 +1,43 @@
+## Cualidades de la Plataforma
+Se refiere a **_cómo_** la plataforma y sus capacidades funcionan y qué se puede esperar en términos de requisitos multifuncionales, requisitos no funcionales o simplemente “-idades”. Ejemplos incluyen la fiabilidad o el rendimiento de los servicios gestionados que pueden medirse con Service Level Objectives (SLOs), la seguridad que puede medirse con el tiempo de mitigación de los riesgos identificados, o la observabilidad que se puede utilizar tanto para depurar como para informar sobre el uso de la plataforma. A menudo se confunden las cualidades con las capacidades, ya que algunos conceptos, como la observabilidad, que se pueden ofrecer como una capacidad (por ejemplo, el Operador OTel proporcionado para recopilar la telemetría de la aplicación) y una calidad declarada (por ejemplo, métricas de la plataforma para medir y alertar sobre el tiempo de actividad de ese Operador OTel proporcionado).
 
-## Cognitive load
-A quantification of the mental costs to a user before they can benefit from a platform capability. Within the umbrella term cognitive load there are actually three types of load: germane, intrinsic, and extraneous. Organizations are healthiest when platforms enable users to focus on the germane load (job-specific problem solving) challenges while simplifying the intrinsic (onboarding of new information or processes to complete their task) and minimizing the extraneous (distractions from the focused task, sometimes nicknamed “[yak shaving](https://en.wiktionary.org/wiki/yak_shaving#:~:text=yak%20shaving%20(uncountable),to%20solve%20a%20larger%20problem.)”) loads.
+## Carga cognitiva
+Una cuantificación de los costos mentales para un usuario antes de que puedan beneficiarse de una capacidad de la plataforma. Dentro del término general de carga cognitiva, en realidad hay tres tipos de carga: germana, intrínseca y extrínseca. Las organizaciones son más saludables cuando las plataformas permiten a los usuarios centrarse en los desafíos de carga germana (resolución de problemas específicos del trabajo) mientras simplifican la carga intrínseca (incorporación de nueva información o procesos para completar su tarea) y minimizan la carga extrínseca (distracciones de la tarea enfocada, a veces apodada “[yak shaving](https://en.wiktionary.org/wiki/yak_shaving#:~:text=yak%20shaving%20(uncountable),to%20solve%20a%20larger%20problem.)”).
```

Comment:
Don't know if "germana" is the accurated word here. Maybe let the original English word?